src/index.js

```diff
@@ -1,8 +1,16 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { createStore, applyMiddleware } from 'redux';
+import { Provider } from 'react-redux';
+import rootReducer from './reducers';
+import thunk from 'redux-thunk';
 import App from './components/App';
 
+export const store = createStore(rootReducer, applyMiddleware(thunk));
```

Comment:
You can add more middlewares here :)

Enhancers, for example.

One of the most common enhancers is `Redux Devtools`

You can add this enhancer this way: 

```
import { compose } from 'redux';
...
const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;

export const store = createStore(rootReducer, composeEnhancers(applyMiddleware(thunk)));
```

Don't forget to install Redux Devtools extension ;)
 [https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd?hl=en](https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd?hl=en)